### PR TITLE
Drop the contao/polyfill-symfony package

### PR DESCRIPTION
**Pull request by leofeyer**
Now that we are on Symfony 5 only, we no longer need `contao/polyfill-symfony`.

### CODE REVIEW

**Comment by aschempp** [comment]
Do we need it again with Symfony 6 maybe? Should we keep it or re-add it then?

**Comment by leofeyer** [comment]
Add it again if necessary.

**Review by aschempp — APPROVED** [review]

File: core-bundle/src/Image/Studio/Figure.php [hunk 1]
```diff
@@ -60,7 +60,6 @@ class Figure
      * All arguments but the main image result can also be set via a Closure
      * that only returns the value on demand.
      *
-     * @param ImageResult                                                                 $image          Main image
```
Comment (bytehead) [inline core-bundle/src/Image/Studio/Figure.php:63]: How is this related?

Comment (leofeyer) [reply core-bundle/src/Image/Studio/Figure.php:63]: This change was made by the CS fixer.
